### ISSUE TITLE
Cannot install library definitions

### ISSUE THREAD

**Issue by cameron-martin** [flowtype/flow-typed#347]
When I try to install a package using flow-typed, I get an error. It looks like a valid version is available (classnames_v2.x.x/flow_v0.28.x-/), so it should not error.

OS: Windows 10.

```
$ flow-typed version

flow-typed@2.0.0-beta.9

$ flow-typed install -f 0.32 classnames@2.2.5                                                                          
                                                                                                                               
• rebasing flow-typed cache...done.                                                                                            
• Installing 1 libdefs...                                                                                                      
  !! Failed to install classnames into C:\Users\camer\Sources\SetupWizard\flow-typed\npm\classnames_v2.x.x.js                  
  ERROR: Unable to find version information for 'classnames_v2.x.x/flow_>=v0.28.x'!                                            
• The following installed libdefs are compatible with your dependencies, but may not include all minor and patch changes for yo
ur specific dependency version:                                                                                                
                                                                                                                               
  • libdef: classnames_v2.x.x (satisfies classnames@2.2.5)                                                                     
                                                                                                                               
  Consider submitting a versioned update for this package to                                                                   
  https://github.com/flowtype/flow-typed/
```

**Comment by jeffmo** [comment]
This appears to be a windows-specific issue. I'm able to repro on a windows machine, but not on my mac.

Windows fixes only fairly recently landed, so there's a good chance we missed something. I'll dig into this today, thanks for the report!

**Comment by jeffmo** [comment]
Just an update: I'm testing and looking into windows issues today.

It's taking a little time to ramp up on windows (not a very familiar environment for me), but I'm hoping to have something by EOD

**Comment by jeffmo** [comment]
Just published `flow-typed@2.0.0-beta.10` which should fix this.
Let me know if you hit any more issues!
